src/components/ApplicationLayout/AppNavigationBar/AppNavigationBar.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import type { PropsWithSpread } from "types";
+import classNames from "classnames";
+import type { HTMLProps, PropsWithChildren } from "react";
+
+type Props = PropsWithSpread<PropsWithChildren, HTMLProps<HTMLDivElement>>;
```

Comment:
```suggestion
type Props = PropsWithSpread<PropsWithChildren, HTMLProps<HTMLHeadElement>>;
```
Not sure on this one though, might be the right one to use?

Reply:
Oops, thanks. `HTMLHeadElement` is for the document `<Head>` element, but it looks like `HTMLElement` is the right one here.